doc/apiv3-documentation.apib

```diff
@@ -1608,7 +1608,6 @@ Note that regular expressions used in the API follow the rules of [Ruby Regular
     + Body
 
             {
-                "_type": "Schema",
```

Comment:
Shouldn't we write what you wrote as a preamble to this PR into the documentation as well?